crates/loader/src/local/assets.rs

```diff
@@ -260,3 +289,36 @@ fn is_absolute_guest_path(path: impl AsRef<Path>) -> bool {
     // paths specifically using Unix logic rather than the system function.
     path.as_ref().to_string_lossy().starts_with('/')
 }
+
+/// Convert strings to glob patterns
+pub fn convert_strings_to_glob_patterns<T: AsRef<str>>(files: &[T]) -> Result<Vec<glob::Pattern>> {
```

Comment:
Could this conversion be an internal concern of `assets::collect`?  It feels like we always need to call them together and I'm wondering if we'd be better off passing the raw strings to collect and having it do the conversion...?  (I do like that the input has been validated before entry though, so I'm in two minds.)

Reply:
Yes, I updated it. Other components only need to call `assets::collect` without getting `exclude_patterns` first. Thanks for the suggestion!